momentum/4/config/ref-rollup-mx-name.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "rollup_mx_name"
+description: "If the associated rollup_mx option is set to 'true', rollup_mx_name specifies the MX host domain to route the traffic to. Valid in host, binding, and binding_group scopes. Usable in Host stanzas, which can be included in global, binding or binding_group scopes."
```

Comment:
This description does not actually show up on the page, because it is in this description area where the title is. If you want this text to display, it needs to be below the --- in one of the below sections.  I think we would want this entire text in the description area below, and this section would actually be the first few sentences of the description below.

Reply:
I know.  From the docs it appears taht that "description" is used by the search function.  The description further down has the actual entry.